core/interfaces/i_selectable_toolbox_item.ts

```diff
@@ -61,3 +61,11 @@ export interface ISelectableToolboxItem extends IToolboxItem {
    */
   onClick(_e: Event): void;
 }
+
+/**
+ * Type guard that checks whether an IToolboxItem is an ISelectableToolboxItem.
+ */
+export function isSelectableToolboxItem(toolboxItem: IToolboxItem):
+    toolboxItem is ISelectableToolboxItem {
```

Comment:
![i-dont-understand-is-it-modern](https://user-images.githubusercontent.com/13686399/216469608-f13c95c3-1aaf-4259-9b50-573de005bfac.gif)

(I'll go look at the typescript docs--I know I've skimmed pass this section before.)